packages/dx-react-demos/src/demo-sources/core-action/bootstrap3/parameterized-action.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
```

Comment:
Why this demo and other demos are in the `./bootstrap3/.` folder?

Reply:
It is the simplest solution currently. We don't see any reason to create "default" theme.